README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# hw01
+## About Me
+HELLO! My name is **Jarret Angbazo** and I am a second-year MPP student at the *Harris School of Public Policy*. I was born in Nigeria and before moving to Chicago for graduate school, I grew up in Maryland. Chicago is the furthest west on the continental United States that I've lived.
+
+Quick two truths and a lie:
+
+1. I did this assignment in 5 minutes
+
+2. I don't hate the player, I hate the game
+
+3. I ran a personal best in the 5k this weekend
+
+Here is a [**link**](https://www.linkedin.com/in/jarretangbazo) to my LinkedIn page. There you will be able to find my employment and education histories.
+
+Thanks for stopping by.
+
+### Process
+To get the changes onto Readme.md, I essentially followed the instructions from the course webpage [**Using Git within RStudio**](https://cfss.uchicago.edu/setup/git-with-rstudio/). Initially, everything did not install correctly so I had to manually change the Git executable path from **/usr/bin/git** to **/usr/local/bin/git**. I then followed the steps for forking and cloning a repository, making edits to the readme file, saving and pushing back to GitHub.
+
+### Reflection on assignment
+To complete this assignment, I used the following [**link**](https://www.markdownguide.org/cheat-sheet/) as a resource for the markdown syntax.
+
+I still feel fairly uncomfortable with Github and perhaps this is the main reason why I am taking this class. The process of forking, cloning, pulling, and pushing seems very complicated. While I am aware that this is mostly due to the terminology associated with GitHub and I'm certain in time this should get easier, it all seems very opaque. Even though I've completed this assignment, I'm not completely sure that I've understood what I've done completely.
```

Comment:
I was also unsure about the entire process but the lab sessions really helped! I think the reflections and seeing what others have done helps too.

Reply:
Practice makes progress. By the end of the quarter you'll feel more comfortable